skimage/feature/tests/test_blob.py

```diff
@@ -1,17 +1,20 @@
-import pytest
+import math
 
 import numpy as np
+import pytest
+from numpy.testing import assert_almost_equal
+
+from skimage._shared import testing
```

Comment:
I recommend directly using `pytest` instead of the aliases defined in `skimage._shared.testing` :wink:

Reply:
Agreed.  We did prefer pytest in another PR recently, but I hadn't updated this one.